Gaijinizer patch/Map063.txt

```diff
@@ -302,166 +330,180 @@ switch?#MTLed
  地下街の電力供給は未だにアクティブになっている。
  天井の配線が無事なら、灯りが付くかもしれない。
 > CONTEXT: Map063/events/7/pages/0/7/Dialogue/yuuma
-It's like a light switch. Underground street
-electricity supply is still active.
-If the wiring of the ceiling is safe, it may be
-lighted.#MTLed
+Looks like a light switch. The underground mall is
+still being supplied with electricity.
+
+If the wiring in the ceiling's still good, you
+could switch on the lights.#Ron
```

Comment:
Remove the line break here, or add another one to push the second set of text cleanly into the new box.